src/index.ts

```diff
@@ -127,8 +127,9 @@ export const create = async (
  */
 export const clearFirst = async (input: HTMLElement) => {
   const container = getReactSelectContainerFromInput(input);
-  // The "clear" button is the first svg element that is hidden to screen readers
-  const clearButton = container.querySelector('svg[aria-hidden="true"]')!;
+  // The "clear" button is constructed from the user-defined `${classNamePrefix}__multi-value__remove`.
+  // This is built from the internal util `cx` of react-select, so we take advantage of the attribute selector here.
+  const clearButton = container.querySelector('[class*="multi-value__remove"]')!;
```

Comment:
We need to keep backwards compatibility with older react-select versions.

What would be the best way to achieve this?
Do we need to default to the svg selector of the new one doesn’t match anything?